common/src/main/java/tachyon/conf/TachyonConf.java

```diff
@@ -308,6 +316,10 @@ public long getBytes(String key, long defaultValue) {
     }
   }
 
+  public long getBytes(String key) {
+    return getBytes(key, 0L);
+  }
+
```

Comment:
For the defaults in these methods, maybe its better to just throw an exception because it seems like the user would never want to run their system with the default value.